cmd/zoekt-sourcegraph-indexserver/sg.go

```diff
@@ -37,20 +38,17 @@ type SourcegraphListResult struct {
 	// Note: this has a side-effect of setting a the "config fingerprint". The
 	// config fingerprint means we only calculate index options for repositories
 	// that have changed since the last call to IterateIndexOptions. If you want
-	// to force calculation of index options use Sourcegraph.GetIndexOptions.
+	// to force calculation of index options use
+	// Sourcegraph.ForceIterateIndexOptions.
 	IterateIndexOptions func(func(IndexOptions))
 }
 
 type Sourcegraph interface {
```

Comment:
Since this is exported, should we add a docstring?

Reply:
I'll add one. this is a main pkg so I am exporting it more for signalling its a type than for outside use. However, we should move most of this code into a pkg and then I imagine this type still being exported.